src/server/src/modules/message.ts

```diff
@@ -208,4 +208,42 @@ export const getConversationList = async (userId: number) => {
             conversation2.latestMessage.date.getTime() - conversation1.latestMessage.date.getTime());
 
     return getConversationListResult;
+};
+
+/**
+ * @description Returns an user object with the the email to initiate a messag.
```

Comment:
Returns a user object with the email to initiate a message.

Reply:
Done.